core/src/test/resources/test.conf

```diff
@@ -67,7 +67,9 @@ primitives.foo {
 
 primitives.bar {
   type: set
-  cache.enabled: true
```

Comment:
guess this syntax might work too.

Reply:
it works in Typesafe Config

Reply:
Should change these back. I'd like to keep them as examples for users. This test passes fine.

Reply:
changed this one back, pretty sure the other one is broken and it's nice to show different syntax imo.